bpf/lbmap/ipv4.go

```diff
@@ -38,6 +38,11 @@ var (
 		int(unsafe.Sizeof(RevNat4Key{})),
 		int(unsafe.Sizeof(RevNat4Value{})),
 		maxEntries)
+	RRSeq4Map = bpf.NewMap("cilium_lb4_rr_seq",
+		bpf.MapTypeHash,
+		int(unsafe.Sizeof(Service4Key{})),
+		int(unsafe.Sizeof(ServiceRRSeq{})),
```

Comment:
RRSeq4Value and RRSeq6Value will be same as ServiceRRSeq, thats why I did not create an interface. If you want me to I can create new types. But why have so many types that are all the same ?